quartz.layout.ts

```diff
@@ -24,7 +24,7 @@ export const defaultContentPageLayout: PageLayout = {
   left: [
     Component.PageTitle(),
     Component.MobileOnly(Component.Spacer()),
-    Component.Search(),
+    Component.Search({ enablePreview: true }),
```

Comment:
```suggestion
    Component.Search({ enablePreview: false }),
```

Reply:
To be consistent with all other components, we shouldn't modify the layout file here. I saw that you already set the `enablePreview: false` in `Search.tsx`, so omitting it here and leaving it as `Component.Search()` is fine.

Probably a good idea to also update the search docs to explain that this config option/feature exists as well, then people can just set it true when they want to use it